helix-term/src/commands.rs

```diff
@@ -1067,6 +1072,13 @@ mod cmd {
             fun: later,
             completer: None,
         },
+        Command {
+            name: "wq",
+            alias: None,
```

Comment:
```suggestion
            name: "write-quit",
            alias: Some("wq"),
```
Let's follow kakoune on this, they did it quite well.

Reply:
I was going to suggest the same :)

Reply:
That's what I've been thinking too, updated.